pkg/pillar/cmd/zedagent/handlecontent.go

```diff
@@ -96,6 +96,25 @@ func parseContentInfoConfig(ctx *getconfigContext,
 		contentConfig.MaxDownloadSize = cfgContentTree.GetMaxSizeBytes()
 		contentConfig.DisplayName = cfgContentTree.GetDisplayName()
 		contentConfig.CustomMeta = cfgContentTree.GetCustomMetaData()
+		contentConfig.IsLocal = true
+		controllerDNID := cfgContentTree.GetDesignatedNodeId()
+		// If this node is not designated node id set IsLocal to false.
+		// Content will be downloaded to only to the designated node id of that content tree.
+		// So on other nodes in the cluster mark the content tree as non-local.
+		// On single node eve either kvm or kubevirt based, this node will always be designated node.
+		// But if this is the contenttree of a container, we download it to all nodes of the cluster,
+		// so set IsLocal true in such case.
+
```

Comment:
The spaces between comments are not consistent here and in other parts of this PR. For example, throughout the code we have the following pattern:

```go
go.Code()

// Comment regarding the next line
go.NextLine()
```

or something like this:

```go
go.Code()

// Comment regarding the next block of code

go.Code()
go.Code()
go.Code()
go.Code()
```